src/hooks.tsx

```diff
@@ -6,20 +6,42 @@ import FlowplayerComponent from "./flowplayer";
  * Hook to use Flowplayer API in an async way - tries to solve the chicken/egg problem
  */
 export const useFlowplayer = ({ token } : { token: string }) => {
-    const ref = useRef(document.createElement("div"))
+    const ref = useRef<HTMLDivElement>(null)
+    
+    // Store flowplayer instances to a state value in order to force re-renders when new instances are available
+    const [flowplayerInstances, setFlowplayerInstances] = useState((flowplayer ? flowplayer.instances : []) || [])
```

Comment:
I'm not sure I understand why this complex ternary is necessary since we are importing `flowplayer` directly, it will **always** be truthy?  Shouldn't it just be another `slice()` of the instances as the initial value?